### ISSUE TITLE
ES2020+ features broken by old rollup dependency

### ISSUE THREAD

**Issue by andylizi** [extend-chrome/rollup-plugin-chrome-extension#112]
### Google search terms

Doesn't apply.

### Describe the bug

Write this:
```js
console.log(null ?? "default")
```
and rollup outputs:
```js
console.log(null );
```

### How do we reproduce?

1. Put any statement with a [nullish coalescing operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator) into a content script or whatever.
2. If using Typescript or Babel, set the target to `esnext` to avoid desugaring.
3. Build.

### Expected behavior

```js
console.log(null ?? "default");
```

### Actual behavior

```js
console.log(null );
```

### Please complete the following information:

- Library Version: 3.6.4
- Operating System: Windows 10
- Browser: Chrome 94
- Node Version: 16.8.0

### Cause

The problem is

https://github.com/extend-chrome/rollup-plugin-chrome-extension/blob/2a5623a2938254e902a1089f85946cb57ec7fd0b/package.json#L48

which is used in [`mixed-format/regenerateBundle`](https://github.com/extend-chrome/rollup-plugin-chrome-extension/blob/v3.6.2/src/mixed-format/regenerateBundle.ts#L46-L65). This ancient version of rollup (released in March 2020) can't handle nullish coalescing operators (ES2020 was approved on June 2020) and will somehow remove them in the process.

### Workaround

```diff
  "scripts": {
    "build": "rollup -c",
+   "postinstall": "rm -rf node_modules/rollup-plugin-chrome-extension/node_modules/rollup"
  },
```

This lets the Node resolver to use the installed rollup from the project itself, which hopefully won't be `1.x.x`.

### Solution

Upgrade rollup to v2 and move it to [peer dependencies which was introduced exactly for this kind of purpose](https://nodejs.org/es/blog/npm/peer-dependencies/).

**Comment by jacksteamdev** [comment]
@andylizi Thanks for bringing this to my attention! Do you have time to make a PR?

**Comment by jacksteamdev** [comment]
@andylizi This should be fixed in the beta release, see #117
